### PR TITLE
Documentation: Add link on the provisional page to WET-BOEW provision…

### PR DESCRIPTION
**Pull request by GormFrank**
…al features

+ Added SRI for v7.0.1

### CODE REVIEW

**Comment by GormFrank** [comment]
Pre-approved during technical review as long as the href has the right value...

File: site/pages/provisional-en.hbs [hunk 1]
```diff
@@ -185,6 +185,8 @@
 </tbody>
 </table>
 
+<p>For more provisional features, check out the <a href="">WET-BOEW provisional features page</a>.</p>
```
Comment (duboisp) [inline site/pages/provisional-en.hbs:188]: missing href value

Comment (GormFrank) [reply site/pages/provisional-en.hbs:188]: I know I pointed it out in my comment...

**Comment by GormFrank** [comment]
Took advantage of this PR to also add SRI to GCWeb v7.0.1 release notes

**Pull request merged**